test/metabase/query_processor_test/nested_queries_test.clj

```diff
@@ -242,6 +242,19 @@
                       {:aggregation [:count]
                        :breakout    [$price]}))))))))))
 
+(deftest grouped-expression-in-card-test
```

Comment:
It's not a bad idea to put `testing` context in here with the original GitHub issue number. That way if there's a regression we can find the test that was supposed to catch it, or if there's a test we don't understand we can jump back to the original GitHub issue if we need more context

Reply:
Added the reference.